ui/src/app/components/webcomponents/dry-run-form.component.ts

```diff
@@ -225,25 +224,26 @@ export class DryRunFormComponent extends BaseComponent implements OnInit {
   }
 
   setConfigurationId(configuration: AdhocRunConfiguration) {
-    this.refresh= false;
+    this.refresh = false;
     this.configuration = undefined;
-    if(configuration?.id) {
+    if (configuration?.id) {
       this.configuration = configuration;
-      if(configuration!=null && configuration.platformOsVersionId!=null) {
+      if (configuration != null && configuration.platformOsVersionId != null) {
         this.platformService.findOsVersion(configuration.platformOsVersionId, this.testPlan.testPlanLabType).subscribe((platformOsversion) => {
           configuration.platform = platformOsversion.platform;
           this.setDryFormValues(configuration);
         });
-      }
-      else{
+      } else {
         this.setDryFormValues(configuration);
       }
     } else {
       this.configuration = this.noneConfiguration;
       this.createExecution();
       this.addFormControls();
     }
-    setTimeout(()=> {this.refresh=true}, 100);
+    setTimeout(() => {
```

Comment:
why setTimeOut required here? @manikanta-reddy-t

Reply:
for smooth transition between changes in dry run mobile native form